.github/workflows/go.yml

```diff
@@ -91,6 +92,7 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage-fixed.txt
+          flags: integration-tests,${{ matrix.artifactory }},${{ runner.os }}
```

Comment:
Maybe remove the matrix.artifactory? Is it enough to separate by unit and integration with the total (Linux)?

Reply:
Agreed